conformance/tests/httproute-redirect-port copy.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: gateway.networking.k8s.io/v1beta1
```

Comment:
Is this filename `conformance/tests/httproute-redirect-port copy.yaml` correct?

Reply:
Fixed. Thanks for catching this!